index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microfolio</title>
+    <link
+      rel="shortcut icon"
+      href="https://img.icons8.com/external-others-inmotus-design/67/null/external-B-alphabet-others-inmotus-design-22.png"
+      type="image/x-icon"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav>
+      <img
+        id="menu"
+        src="https://img.icons8.com/external-tal-revivo-regular-tal-revivo/96/null/external-mobile-application-hamburger-menu-setting-interface-basic-regular-tal-revivo.png"
+      />
+    </nav>
+    <img id="i1" src="imgs/i1.png" alt="illustration" />
+    <section id="content-1">
+      <h1>
+        <span class="clr-blue">Hey there. <br />I'm Blesslin <br /></span>
+        <span class="clr-green"> I'm a Software <br />Developer</span>
+      </h1>
+      <p class="content-1-p">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitate to contact me.
+      </p>
+      <div class="flex">
+        <img src="imgs/socialmedia.png" alt="social media" />
+      </div>
+      <img
+        id="d"
+        src="https://img.icons8.com/external-dreamstale-lineal-dreamstale/32/null/external-down-arrow-arrows-dreamstale-lineal-dreamstale-2.png"
+      />
+    </section>
+    <section id="content-2">
+      <h1 class="clr-blue">My Recent <br />Works</h1>
+      <img src="imgs/Indicator.png" alt="indicator" />
+    </section>
```

Comment:
- [ ] Lastly, the background image doesn't show properly in your design as it was in the Figma design, and just as the previous reviewer mentioned, please ensure that you have the right background image so your design looks similar to the original design :+1: 
- Your Design
![mobile (1)](https://user-images.githubusercontent.com/102699958/227289774-5054aebb-2f4e-475a-b227-478311ea65a4.png)
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/227290244-8a913454-a8a9-4c5b-ba23-f3affed1c06e.png)